drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java

```diff
@@ -787,11 +788,19 @@ public void newAttributeOrMetaDataColumn() {
         if ( isReadOnly() ) {
             return;
         }
-        view.newAttributeOrMetaDataColumn();
+        view.newAttributeOrMetaDataColumn( getReservedAttributeNames() );
```

Comment:
Actually the other way around. `existingAttributeNames` should be `reservedAttributeNames`.  This is because reserved has both, the attribute names we block and the ones that already exist.
Still, well spotted.